lib/public/views/Logs/Create/LogCreationModel.js

```diff
@@ -51,7 +53,7 @@ export class LogCreationModel extends Observable {
                 },
             );
         } else {
-            this._parentLog = RemoteData.success(null);
+            this._parentLog = RemoteData.notAsked();
```

Comment:
I strongly disagree in the fact to use the `RemoteData.notAsked()` to define if there is a parent log or not. In my opinion we base the code on an implicit information (if the data has not been asked, it means that there is no reason to ask for it). But this relation may be at some point broken, for example if we delay the loading of the parent log, forget to update it and so on.
The `RemoteData` should only carry information about the **state of a remote information**, but nothing about the information itself, because else we are mixing two information for very different contexts in a variable (the state of the parent and the actual parent in the case it is null), and splitting another information in two variable (the actual parent log that may be null is split between the remote data and its payload)